avoinspectorexample/build.gradle

```diff
@@ -33,15 +35,17 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'androidx.appcompat:appcompat:1.2.0'
-    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
 
     implementation 'com.segment.analytics.android:analytics:4.8.2'
+    implementation project(path: ':avoinspector')
+
+   // implementation 'com.github.avohq.android-avo-inspector:dev:2.0.0'
```

Comment:
Is this left here by mistake?

Reply:
I think this has some value - i.e. how you would include the dependency from the repo. Would add a comment to make it clear.